src/sage/groups/generic.py

```diff
@@ -479,7 +481,7 @@ def bsgs(a, b, bounds, operation='*', identity=None, inverse=None, op=None):
 
     if ran < 30:    # use simple search for small ranges
         d = c
-#        for i,d in multiples(a,ran,c,indexed=True,operation=operation):
+        #        for i,d in multiples(a,ran,c,indexed=True,operation=operation):
```

Comment:
```suggestion
        # for i,d in multiples(a,ran,c,indexed=True,operation=operation):
```
Did you mean this? Not sure about this change either way: I prefer the `#` at the beginning of the line for commenting things out. (This is a matter of taste.)

Reply:
Ah I might have formatted that accidentally *blames editor*. I'll keep the new change since I prefer it :D